keps/sig-windows/20190714-windows-csi-support.md

```diff
@@ -371,7 +379,7 @@ New YAMLs for DaemonSets associated with CSI Node Plugins needs to be authored t
 
 ### Risks and Mitigations
 
-Any pod on a Windows node can be configured to mount `\\.\pipe\csi-proxy-v[N]` and perform privileged operations. Thus csi-proxy presents a potential security risk. To mitigate the risk, here are some options:
+Any pod on a Windows node can be configured to mount `\\.\pipe\csi-proxy-v[N]` and perform privileged operations. Thus csi-proxy presents a potential security risk. To mitigate the risk, some options are described below:
```

Comment:
are there already things that could be mounted into pods that allowed arbitrary hostpath mounts that would allow similar escalations/capabilities? if so, the mere existence of the csi proxy isn't really an increased risk for those pods.

the increased risk comes from the csi-proxy handling of data plumbed to it by the csi drivers from restricted user-specified PVCs/Pods that were not allowed hostPath access, hence the data validation questions in https://github.com/kubernetes/enhancements/pull/1184/files#r308785333

Reply:
Windows doesn't support container mount projection today, so there's no way to create new mounts that can be interpreted by the host.